src/ResourceManager/KeyVault/Commands.KeyVault/Commands.KeyVault.csproj

```diff
@@ -49,7 +51,37 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
   <ItemGroup>
+    <Compile Include="Commands\AddAzureKeyVaultCertificate.cs" />
+    <Compile Include="Commands\SetAzureKeyVaultCertificateAttribute.cs" />
+    <Compile Include="Commands\StopAzureKeyVaultCertificateOperation.cs" />
+    <Compile Include="Commands\GetAzureKeyVaultCertificateOperation.cs" />
+    <Compile Include="Commands\ImportAzureKeyVaultCertificate.cs" />
+    <Compile Include="Commands\AddAzureKeyVaultCertificateContact.cs" />
+    <Compile Include="Commands\AddAzureKeyVaultCertificateIssuer.cs" />
+    <Compile Include="Commands\GetAzureKeyVaultCertificate.cs" />
+    <Compile Include="Commands\GetAzureKeyVaultCertificateContact.cs" />
+    <Compile Include="Commands\GetAzureKeyVaultCertificateIssuer.cs" />
+    <Compile Include="Commands\GetAzureKeyVaultCertificatePolicy.cs" />
+    <Compile Include="Commands\NewAzureKeyVaultCertificateAdministratorDetails.cs" />
```

Comment:
'New' I think is not the right verb because you are not creating a new resource according to this: https://msdn.microsoft.com/en-us/library/ms714428(v=vs.85).aspx
Can we use set for both update and creating of AdministratorDetails, CertificatePolicy, CertificateSigningRequest and OrganizationDetails?